res/TensorFlowLiteRecipes/Add_STR_000/test.recipe

```diff
@@ -1,33 +0,0 @@
-# This is test for import/export of STRING tensortype
-# interpreter or runtime may fail as Add won't support this
-
-operand {
-  name: "ifm"
-  type: STRING
-  shape { }
-}
-operand {
-  name: "suffix"
-  type: STRING
-  shape { }
-  filler {
-    tag: "explicit"
-    arg: "Hello"
-  }
-}
-operand {
-  name: "ofm"
-  type: STRING
-  shape { }
-}
-operation {
-  type: "Add"
```

Comment:
@seanshpark I'd like to remove `Add_STR_000` and `Add_STR_001`, because their recipes are invalid.

`Add` operator does not support STRING inputs both in circle and tflite. It seems that this recipe has to be re-written with CUSTOM Op (PTAL https://github.com/Samsung/ONE/issues/6730#issuecomment-833415744).